common/lib/protocol-definitions/api-report/protocol-definitions.api.md

```diff
@@ -40,6 +41,17 @@ export interface IAttachment {
     id: string;
 }
 
+// @public
+export interface IAudience extends EventEmitter {
```

Comment:
I more than open to suggestions about how to deal with this. I'm adding it to the protocol-definitions as it's the lowest common ancestor of all packages who would need it, in the context of NOT adding a dependency between container definitions and protocol base (which should not be done IMO).

Reply:
This is a non-issue now, this can be left here, as the custom protocol handler which needs an audience has been brought into the loader for now